README.adoc.in

```diff
@@ -1,7 +1,7 @@
-= Application Templates for Red Hat Single Sign-On 7.3 continuous delivery (CD) Tech Preview OpenShift container image
+= Application Templates for Red Hat Single Sign-On 7.3 OpenShift container image
 
 This project contains OpenShift v3 application templates which support applications based on JBoss Middleware products.
-Source can be found https://github.com/jboss-container-images/redhat-sso-7-openshift-image/tree/sso-cd-dev[here].
+Source can be found https://github.com/jboss-container-images/redhat-sso-7-openshift-image/tree/sso-73-dev[here].
```

Comment:
This should be `Source can be found https://github.com/jboss-container-images/redhat-sso-7-openshift-image/tree/sso73-dev` (the current one returns HTTP 404 Not Found).